relayer/core/relayV2.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Snowfork
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package core
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/snowfork/polkadot-ethereum/relayer/workers"
+	"github.com/snowfork/polkadot-ethereum/relayer/workers/ethrelayer"
+)
+
+type RelayV2 struct {}
+
+func (re *RelayV2) Run() error {
+	config, err := LoadConfig()
```

Comment:
Inline with the new independent worker approach, we should probably reorganise our configuration hierarchy to look like this:

```toml
[core]
foo = 1
bar = 2

[workers]
    [workers.EthRelay]
        [workers.EthRelay.ethereum]
        address = "http://foo.bar"

        [workers.EthRelay.substrate]
        address = "http://foo.bar"

    [workers.ParaRelay]
        [workers.ParaRelay.ethereum]
        address = "http://foo.bar"

        [workers.ParaRelay.substrate]
        address = "http://foo.bar"
```

Then the config for an individual worker gets parsed into a `Config` object and passed to the worker initializer or factory function.

Reply:
I think config reorganization should go in another PR because it will require changes in listener / writer logic, etc. This PR is just for the worker wrappers

Reply:
That's fine. For now perhaps, I guess you could probably pass the entire config down to the worker factory and then the worker can pick out the parts it wants.